llama-index-integrations/embeddings/llama-index-embeddings-fireworks/llama_index/embeddings/fireworks/utils.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional, Tuple
+
+from llama_index.core.llms.generic_utils import get_from_param_or_env
```

Comment:
Should be `from llama_index.core.base.llms.generic utils import get_from_param_or_env`